hydra_base/lib/network.py

```diff
@@ -2878,13 +2774,13 @@ def clone_network(network_id,
                                                         f"{new_network_name}%")).all()
 
     if len(ex_network) > 0:
-        new_network_name = new_network_name + " " + str(len(ex_network))
+        new_network_name = f"{new_network_name}  ({str(len(ex_network))})"
```

Comment:
This change breaks the test `test_clone_network_into_existing_project()` in `test_network.py` because the `assert` on line 1029 is comparing the cloned and original network names.  The clone's name now includes parentheses.